justfile

```diff
@@ -0,0 +1,40 @@
+set dotenv-load
+
+# After configuring your `.env` file, run `just deploy` for a dry run, or `just deploy '--broadcast'` for the live deploy.
+deploy flags='':
+    #!/usr/bin/env bash
+    set -euo pipefail
+
+    echo "COMMIT HASH: $(git rev-parse HEAD)" >> deploys.txt
+
+    # Save deploy addresses to `deploys.txt`
+    just __deploy | grep -E '^(== Logs ==|  COMMIT HASH|  FOUNDRY_PROFILE|  CHAIN ID|  CHAIN NAME|  CounterBasic|  CounterWithImmutables)' >> deploys.txt
```

Comment:
nit: if you use `| tee -a deploys.txt` instead of `>> deploys.txt` you can show the output on the console *and* append it to the log file.